src/containers/App/App.js

```diff
@@ -10,8 +10,8 @@ import Login from '../Login/Login'
 import { Route, NavLink, withRouter } from 'react-router-dom';
 import SignUp from '../SignUp/SignUp';
 
-class App extends Component {
-
+export class App extends Component {
+  
```

Comment:
I remember this from Christie's lesson, seems redundant but it looks like it helps for testing